ethereum/contracts/@openzeppelin/contracts/token/ERC20/IERC20.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-// OpenZeppelin Contracts (last updated v4.6.0) (token/ERC20/IERC20.sol)
+// OpenZeppelin Contracts (last updated v4.9.0) (token/ERC20/IERC20.sol)
```

Comment:
Strange that the last modified version changed, but we don't see a diff anywhere else in the file.